AxonDeepSeg/visualization/visualize.py

```diff
@@ -1,265 +0,0 @@
-# coding: utf-8
-
-import sys
-from pathlib import Path
-import pickle
-
-# Scientific modules imports
-import numpy as np
-from sklearn.metrics import accuracy_score
-from sklearn import preprocessing
-from tabulate import tabulate
-
-# Graphs and plots imports
-from matplotlib.backends.backend_agg import FigureCanvasAgg as FigureCanvas
-from matplotlib.figure import Figure
-
-# AxonDeepSeg imports
-import AxonDeepSeg.ads_utils as ads
-from AxonDeepSeg.ads_utils import convert_path
-from AxonDeepSeg.testing.segmentation_scoring import score_analysis, dice
-
-
-def visualize_training(path_model, iteration_start_for_viz=0):
-    """
-    :param path_model: path of the folder with the model parameters .ckpt
-    :param iteration_start_for_viz: first iterations can reach extreme values,
-        iteration_start_for_viz set a beginning other than epoch 0
-    :return: matplotlib.figure.Figure
-
-    The returned figure represents the evolution of the loss and the accuracy 
-    evaluated on the validation set along the learning process.
-    If the learning began from an initial model, the figure plots first the
-    accuracy and loss evolution from this initial model and then stacks the
-    evolution of the model.
-    """
-
-    # If string, convert to Path objects
-    path_model = convert_path(path_model)
-
-    def _create_figure_helper(data_evolution):
-        fig = Figure()
-        FigureCanvas(fig)
-
-        # Drawing the evolution curves
-        ax1 = fig.subplots()
-        ax2 = ax1.twinx()
-        ax1.plot(
-            data_evolution["steps"][iteration_start_for_viz:],
-            data_evolution["accuracy"][iteration_start_for_viz:],
-            "-",
-            label="accuracy",
-        )
-        ax1.set_ylim(ymin=0)
-        ax2.plot(
-            data_evolution["steps"][iteration_start_for_viz:],
-            data_evolution["loss"][iteration_start_for_viz:],
-            "-r",
-            label="loss",
-        )
-
-        # Annotating the graph
-        ax1.set_title("Accuracy and loss evolution")
-        ax1.set_xlabel("Epoch")
-        ax1.set_ylabel("Accuracy")
-        ax2.set_ylabel("Loss")
-        return fig
-
-    evolution = retrieve_training_data(path_model)
-    fig = _create_figure_helper(evolution)
-
-    return fig
-
-
-def visualize_segmentation(path):
```

Comment:
I think I would keep the `visualize_segmentation` function in the code for now and only remove the other functions related to training.

Similar to the functions of the other files in the visualization folder, it look like a useful tool but is not used at the moment. I would keep it for now, but we should re-assess its usage later in the integration process with ivadomed.

Reply:
Done in be539d3